spec/models/post_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
Hi @TedLivist , thanks for your review. 

Could you kindly specify what tests should I write for `Comment` and `Like`? 
I think only association tests are possible in these classes since there is no validation or method.